packages/manager/src/components/application/ApplicationIndex.tsx

```diff
@@ -13,10 +16,26 @@ const styles = StyleSheet.create({
   },
 });
 
-export default () => {
+export default withRouter(({ history }) => {
```

Comment:
Since we already use hooks in most of components.
It should use the `useHistory` hook to replace this

https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/hooks.md#usehistory